Utilities/bootstrap

```diff
@@ -832,6 +850,8 @@ def main():
                         default=["/usr/local"], metavar="PATHS")
     parser.add_argument("-v", "--verbose", action="store_true",
                         help="use verbose output")
+    parser.add_argument("--swift-build-dir", dest="swift_build_dir",
```

Comment:
The purpose of this parameter might get a little confusing given we also use an env variable `SWIFT_BUILD_PATH`.

Reply:
I agree; and that's a different purpose, right?  i.e. it's the path of the `swift-build` executable, not the path into which Swift was built...  is that correct?

Reply:
`swift_build_dir` is to point to, e.g., `.../build/Ninja-DebugAssert/swift-linux-x86_64`. The name `swift_build_dir` is consistent across the various `apple/swiftXYZ` projects. 

Though if you have a suggestion for an alternative name I'm all ears.

Reply:
I see. I think we should rename SWIFT_BUILD_PATH to SWIFTPM_BUILD_DIR then (in its own PR).